packages/react-reconciler/src/ReactFiberWorkLoop.new.js

```diff
@@ -689,37 +689,32 @@ function ensureRootIsScheduled(root: FiberRoot, currentTime: number) {
   markStarvedLanesAsExpired(root, currentTime);
 
   // Determine the next lanes to work on, and their priority.
-  const newCallbackId = getNextLanes(
+  const nextLanes = getNextLanes(
```

Comment:
Looks like the comment on line 681 (which references the expiration time) is out of date, should we update that as well?